org_permissions/models.py

```diff
@@ -34,4 +34,12 @@ class Meta:
         verbose_name_plural = 'Organization permissions'
 
     def __str__(self):
-        return '{} : {}'.format(str(self.organization), ' : '.join(str(perm) for perm in self.permissions.all()))
+        return '{}'.format(str(self.name))
+
+class OrganizationStaffGroup(Group):
```

Comment:
What's the purpose of OrganizationStaffGroup?
Why just using Django built-in Group is not enough?